dashboard/app/models/pd/enrollment.rb

```diff
@@ -8,16 +8,19 @@
 #  email              :string(255)      not null
 #  created_at         :datetime
 #  updated_at         :datetime
-#  code               :string(255)
 #  school             :string(255)
+#  code               :string(255)
```

Comment:
It's not actually changing position in the schema. This annotation was incorrect. Since I re-annotated, it switched back to the actual ordering. It was switched accidentally in https://github.com/code-dot-org/code-dot-org/pull/8394 and fixed (in the schema, not the annotation) in https://github.com/code-dot-org/code-dot-org/commit/5f9894e1391854ce7e495f571c5ad1c2b27b61b3